src/routes/controllers/users.controller.js

```diff
@@ -189,3 +189,53 @@ export async function getUsersBookmarks(req, res) {
     return errorResponse(res, 500, error.message);
   }
 }
+
+/**
+ *
+ * @param {Object} req express request
+ * @param {Object} res express response
+ * @returns {Array} user likes
+ */
+export async function getAllArticlesLikedByUser(req, res) {
```

Comment:
Nice work man! Since this function returns either users likes or articles we can just name this function `getAllUserLikes`.

Reply:
Thank you, I will implement this feedback ASAp

Reply:
resolved